### PR TITLE
Fixing Links from Clara's Scanner

### PR DESCRIPTION
**Pull request by sweeneyskirt-sl**
### Description
Fixed a lot of links that either pointed to the wiki or were broken due to a missing front `/`.

### Motivation and Context
@claramyko created a scan that identifies our broken links and links that point to the wiki.

### Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [X] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/dev/glossary.md [hunk 1]
```diff
@@ -255,7 +255,7 @@ See: _[native app](#native-app), [hybrid app](#hybrid-app)._
 
 ### Native App
 
-A mobile software app written in a programming language specific to the platform it is being developed for: either iOS or Android. More information: [Live Testing for Native Mobile Apps on Real Devices](mobile-apps/live-testing/live-mobile-app-testing), [Mobile App Testing with Espresso and XCUITest](mobile-apps/automated-testing/espresso-xcuitest).
+A mobile software app written in a programming language specific to the platform it is being developed for: either iOS or Android. More information: [Live Testing for Native Mobile Apps on Real Devices](/mobile-apps/live-testing/live-mobile-app-testing), [Mobile App Testing with Espresso and XCUITest](mobile-apps/automated-testing/espresso-xcuitest).
```
Comment (spider-sauce) [inline docs/dev/glossary.md:258]:
```suggestion
A mobile software app written in a programming language specific to the platform it is being developed for: either iOS or Android. More information: [Live Testing for Native Mobile Apps on Real Devices](/mobile-apps/live-testing/live-mobile-app-testing), [Mobile App Testing with Espresso and XCUITest](/mobile-apps/automated-testing/espresso-xcuitest).
```

File: docs/mobile-apps/supported-devices.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ If you need...
 
 ### Public Device Cloud
 
-Our public cloud, available to all users regardless of pricing plan, contains a wide selection of thoroughly cleaned devices. They are subject to availability. On the mobile device selection screen, if a device is in use, it'll be [marked with a **Busy** flag](https://docs.saucelabs.com/mobile-apps/live-testing/live-mobile-app-testing#selecting-a-real-device). Here are some use cases:
+Our public cloud, available to all users regardless of pricing plan, contains a wide selection of thoroughly cleaned devices. They are subject to availability. On the mobile device selection screen, if a device is in use, it'll be [marked with a **Busy** flag](mobile-apps/live-testing/live-mobile-app-testing#selecting-a-real-device). Here are some use cases:
```
Comment (spider-sauce) [inline docs/mobile-apps/supported-devices.md:66]:
```suggestion
Our public cloud, available to all users regardless of pricing plan, contains a wide selection of thoroughly cleaned devices. They are subject to availability. On the mobile device selection screen, if a device is in use, it'll be [marked with a **Busy** flag](/mobile-apps/live-testing/live-mobile-app-testing#selecting-a-real-device). Here are some use cases:
```

File: docs/overview.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ hide_table_of_contents: true
     <h2>Automated Testing</h2>
     <p>Browse through the Automated Testing documentation to see examples of how to execute tests in your preferred automation framework.</p>
     <ul>
-        <li><a href="https://wiki.saucelabs.com/display/DOCS/Getting+Started+with+Selenium+for+Automated+Website+Testing">Selenium</a> and <a href="https://wiki.saucelabs.com/display/DOCS/Getting+Started+with+Appium+for+Mobile+Application+Testing">Appium</a></li>
+        <li><a href="/web-apps/automated-testing/selenium">Selenium</a> and <a href="mobile-apps/automated-testing/appium">Appium</a></li>
```
Comment (spider-sauce) [inline docs/overview.md:25]:
```suggestion
        <li><a href="/web-apps/automated-testing/selenium">Selenium</a> and <a href="/mobile-apps/automated-testing/appium">Appium</a></li>
```

File: docs/secure-connections/sauce-connect/setup-configuration/specialized-environments.md [hunk 4]
```diff
@@ -164,4 +164,4 @@ Sauce Headless is a lightweight infrastructure that allows developers to run ear
 For instructions on how to set up Sauce Connect Proxy tunnels with your Sauce Headless tests, see [Getting Started With Sauce Headless](/headless).
 
 ## Legacy RDC (TestObject)
-For instructions on how to run Sauce Connect Proxy on our Legacy RDC platform (TestObject), see [Creating a Sauce Connect Tunnel for Legacy Real Device Cloud](https://wiki.saucelabs.com/display/DOCS/Creating+a+Sauce+Connect+Tunnel+for+Legacy+Real+Device+Cloud).
+For instructions on how to run Sauce Connect Proxy on our Legacy RDC platform (TestObject), see [Creating a Sauce Connect Tunnel for Legacy Real Device Cloud](legacy-tunnels).
```
Comment (spider-sauce) [inline docs/secure-connections/sauce-connect/setup-configuration/specialized-environments.md:167]:
```suggestion
For instructions on how to run Sauce Connect Proxy on our Legacy RDC platform (TestObject), see [Creating a Sauce Connect Tunnel for Legacy Real Device Cloud](/secure-connections/sauce-connect/setup-configuration/legacy-tunnels/).
```

**Review by spider-sauce — CHANGES_REQUESTED** [review]
minor changes, see my suggested commits.

**Comment by sweeneyskirt-sl** [comment]
Thanks for reviewing so quickly, @spider-sauce! Good catches. Hopefully that fixes the build I broke. LOL

**Pull request merged**